tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -16,36 +16,40 @@
 
 :Upstream: No
 """
+from datetime import datetime
+from datetime import timedelta
+
 import pytest
+from nailgun import entities
 
+from robottelo.api.utils import wait_for_tasks
 from robottelo.rh_cloud_utils import get_local_file_data
 from robottelo.rh_cloud_utils import get_remote_report_checksum
+from robottelo.rh_cloud_utils import get_report_data
 
 
-@pytest.mark.tier3
-def test_rhcloud_inventory_e2e(organization_ak_setup, registered_hosts, session):
-    """Generate report and verify its basic properties
+def setting_update(name, value):
+    """change setting value"""
+    setting = entities.Setting().search(query={'search': f'name="{name}"'})[0]
+    setting.value = value
+    setting.update({'value'})
 
-    :id: 833bd61d-d6e7-4575-887a-9e0729d0fa76
 
-    :expectedresults:
+def disable_inventory_settings():
+    setting_update("obfuscate_inventory_hostnames", False)
+    setting_update("obfuscate_inventory_ips", False)
+    setting_update("exclude_installed_packages", False)
```

Comment:
Please switch these to single-quote strings.

Reply:
Updated the pr.